codec-http/src/main/java/io/netty/handler/codec/http/ServerCookieEncoder.java

```diff
@@ -68,6 +68,7 @@ public static String encode(Cookie cookie) {
 
         if (cookie.maxAge() != Long.MIN_VALUE) {
             add(buf, CookieHeaderNames.MAX_AGE, cookie.maxAge());
+            add(buf, CookieHeaderNames.EXPIRES, HttpHeaderDateFormat.get().format(cookie.maxAge() * 1000 + System.currentTimeMillis()));
```

Comment:
@Scottmitch I fixed a few glitches and a test. Yes, dates must be UTC, see [RFC6265](http://tools.ietf.org/html/rfc6265#section-5.1.1).